enterprise/backend/test/metabase_enterprise/transforms_python/models/python_library_test.clj

```diff
@@ -0,0 +1,78 @@
+(ns metabase-enterprise.transforms-python.models.python-library-test
```

Comment:
This seems to mostly test things 1-1 with the API test.

Do we really need both?